src/views/TimezoneModal.vue

```diff
@@ -120,9 +116,10 @@ export default defineComponent({
           // Currently backend API returns some of the legacy timezones which are not found in moment list
           // Due to which if we set them we get an error
           // Filtered them out till it is fixed at backend
-          this.timeZones = resp.data.filter((timeZone: any) => {
```

Comment:
Why we have commented this out?

Reply:
Removed the commented code and comments because we are not using moment anymore.

Reply:
I think here we are checking if the server timezone belongs to moment as well. We should also handle it for luxon. Check if we are similar method in Luxon

Reply:
Handled it for luxon as well